src/os_posix.c

```diff
@@ -354,7 +328,7 @@ os_status_t os_directory_create(
 		if ( result != OS_STATUS_SUCCESS )
 		{
 			os_time_elapsed( &start_time, &time_elapsed );
-			os_time_sleep( LOOP_WAIT_TIME, OS_TRUE );
+			os_time_sleep( 100u, OS_TRUE );
```

Comment:
can you define the 100u so that it is more readable, either local scope or higher?